src/js/views/globals/search/patient-search.scss

```diff
@@ -69,3 +71,48 @@
   padding: 16px 0;
   text-align: center;
 }
+
+.patient-search__tip {
+  border-bottom: 1px solid $black-90;
+  padding-bottom: 16px;
+  padding-left: 60px;
+  text-align: left;
+}
+
+.patient-search__search-by {
+  padding: 16px 0;
+  padding-left: 60px;
+  text-align: left;
+}
+
+.patient-search__search-by-title {
+  font-weight: 600;
+}
+
+.patient-search__search-by-option {
+  margin-top: 8px;
+}
+
+.patient-search__search-by-button {
+  background-color: color(light_blue, light);
+  border: none;
+  color: $black-20;
+  cursor: auto;
+  font-size: 14px;
+  margin-right: 8px;
+  padding: 4px 12px;
+}
+
+.patient-search__search-by-button-label {
+  vertical-align: middle;
+}
+
+.patient-search__shortcut {
+  bottom: 0;
+  left: 0;
+  margin-bottom: 24px;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  width: 100%;
+}
```

Comment:
Moves the keyboard shortcut tip to the bottom of the modal.

![Screen Shot 2022-09-08 at 12 08 36 PM](https://user-images.githubusercontent.com/35355575/189183552-c3d2e92a-5172-49f2-a9de-6d3aa8d4f4f9.png)